onnx/helper.py

```diff
@@ -45,6 +48,8 @@ def make_node(
         node.name = name
     if doc_string:
         node.doc_string = doc_string
+    if domain:
```

Comment:
You need to explicitly check for `None` here, otherwise here will skip setting the domain when it's empty string